components/Article.tsx

```diff
@@ -37,6 +37,7 @@ const Article: React.FunctionComponent<Props> = props => {
     const metaTags: MetaTags = {
         image: props.socialImage ?? props.image,
         description: props.description,
+        type: 'article',
```

Comment:
Going back to that other discussion — do we want this to read `post`? `resource`? `record`? I guess because we are sharing the component for now between different content types, maybe just keeping it in line with the component makes sense. Though I would image that in the future we would have different top-level components for different content types.

Reply:
Good questio and definitely very relevant to the naming question of resources.

I'm not sure if there's any better advice for what are allowed values in `og:type` but I referred to this [section in the Open Graph protocol](https://ogp.me/#no_vertical) website that lists these as the options:

- article
- book
- profile
- website

Reply:
Makes sense! Yeah, `article` seems like the best fit.